src/Razor/src/Microsoft.AspNetCore.Razor.Language/ComponentResources.resx

```diff
@@ -117,6 +117,15 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
+  <data name="AttributeDirective_AttributeToken_Description" xml:space="preserve">
+    <value>The C# attribute that will be applied to the current document.</value>
+  </data>
+  <data name="AttributeDirective_AttributeToken_Name" xml:space="preserve">
+    <value>Attribute</value>
+  </data>
+  <data name="AttributeDirective_Description" xml:space="preserve">
+    <value>Specifies the C# attribute that will be applied to the current document.</value>
```

Comment:
This should say that it applies to the generated class.

Reply:
I said document because that is what we say for `@implements` directive which also applies to the class. Should we change that to say class as well?

Reply:
yeah I think so.